cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
-  from '../support/pages/homeСatalogue.pageObject';
-import faker from 'faker';
+  from '../support/pages/homeCatalogue.pageObject';
+import { faker } from '@faker-js/faker';
 /// <reference types='cypress' />
 
 const contactForm = new ContactFormPageObject();
 const homePage = new HomeAndCataloguePageObject();
 
 const testData = {
   email: faker.internet.email(),
-  name: faker.name.firstName(),
-  message: faker.random.words(),
+  name: faker.person.firstName(),
+  message: faker.word.words(),
```

Comment:
The use of faker.word.words() might generate a message that is too short or not meaningful. Consider specifying a minimum number of words or using a more suitable faker method to ensure the message is appropriate for a contact form.